manifests/07-operator.yaml

```diff
@@ -21,6 +21,11 @@ spec:
       priorityClassName: system-cluster-critical
       nodeSelector:
         node-role.kubernetes.io/master: ""
+      volumes:
+      - name: image-registry-operator-tls
+        secret:
+          optional: true
```

Comment:
Don't forget to remove this, you'll have a bad time debugging with this if the pod starts before the secret is created and TLS isn't getting setup